README.md

```diff
@@ -221,4 +232,4 @@ $ npm run build-prod
 [MIT](./LICENSE).
 
 
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fmatsuyoshi30%2Fharbor.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fmatsuyoshi30%2Fharbor?ref=badge_large)
\ No newline at end of file
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fmatsuyoshi30%2Fharbor.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fmatsuyoshi30%2Fharbor?ref=badge_large)
```

Comment:
I wonder why this line is out as different...

Reply:
Original README.md did not seem to have a line feed at the end of file. vim i used was set to automatically insert newline at the end of file, which seems to have caused the difference 🙇

Reply:
Thanks for the quick response!
I think there should be a newline at the end of the file... Sorry, I'll fix it later after merge it.